general/envsetup/envsetup.go

```diff
@@ -71,8 +83,15 @@ func (ftc *EnvSetupCommand) clearHeadlineMsg() {
 }
 
 func (ftc *EnvSetupCommand) Run() (err error) {
-	ftc.setHeadlineMsg("Just fill out its details in your browser 📝")
-	time.Sleep(8 * time.Second)
+	if ftc.outputFormat == Human {
+		fmt.Println()
+		fmt.Println()
+		fmt.Println(coreutils.PrintTitle("Thank you for installing JFrog CLI! 🐸"))
+		fmt.Println(coreutils.PrintTitle("We'll now set up a FREE JFrog environment in the cloud for you, and configure your local machine to use it."))
+		fmt.Println("Your environment will be ready in less than a minute.")
```

Comment:
Notice that the above messages no longer exist in the *dev* branch, because they were moved to the *jfrog-cli* project. This code needs to be adapted to these changes.